validator/src/main/resources/expected-data-template/java/ec2/asg/aws-sdk-call-trace.mustache

```diff
@@ -0,0 +1,64 @@
+[{
+  "name": "^{{serviceName}}$",
+  "http": {
+    "request": {
+      "url": "^{{endpoint}}/aws-sdk-call$",
+      "method": "^GET$"
+    },
+    "response": {
+      "status": "^200$"
+    }
+  },
+  "aws": {
+    "account_id": "^{{accountId}}$"
+  },
+  "annotations": {
+    "aws_local_service": "^{{serviceName}}$",
+    "HostedIn_EC2_Environment": "^{{platformInfo}}$",
+    "aws_local_operation": "^GET /aws-sdk-call$"
+  },
+  "metadata": {
+    "default": {
+        "otel.resource.ec2.tag.aws:autoscaling:groupName": "^{{platformInfo}}$",
+        "otel.resource.host.id": "^{{instanceId}}$",
+        "otel.resource.host.image.id": "^{{instanceAmi}}$",
+        "otel.resource.host.type": "^t3.micro$",
+        "aws.span.kind": "^LOCAL_ROOT$"
+    }
+  },
```

Comment:
Need to check for `host.name` and `EC2.InstanceId` (or the trace equivalent of those values) in traces as well.

Reply:
Done